content/en/developers/events/dogstatsd.md

```diff
@@ -165,6 +165,35 @@ $statsd->event('An error occurred.',
   );
 {{< /code-block >}}
 
+With the DogStatsD-PHP library you can submit events via TCP directly to the Datadog API. It's slower but more reliable than using the Agent DogStatsD instance since events are forwarded from your application to the Agent using UDP.
+To use this, you must configure the library with your [Datadog API and application keys][1] instead of the local DogStatS instance:
+
+{{< code-block lang="php" filename="event_through_api.php" >}}
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use DataDog\DogStatsd;
+
+$statsd = new DogStatsd(
+    array('api_key' => '<DATADOG_API_KEY>',
+          'app_key' => '<DATADOG_APPLICATION_KEY>',
+     )
+  );
+
+$statsd->event('An error occurred.',
+    array( 'text' => 'Error message',
+           'alert_type' => 'error'
+    )
+  );
+{{< /code-block >}}
+
+**Note**:
+
+* Sending events with this method uses cURL for API requests.
+* You should use a `try`/`catch` code block to avoid warnings or errors on communication issues with Datadog API.
```

Comment:
```suggestion
* You should use a `try`/`catch` code block to avoid warnings or errors on communication issues with the Datadog API.
```
missed this on first pass